README.md

```diff
@@ -21,6 +21,16 @@ lakeFS supports AWS S3 or Google Cloud Storage as its underlying storage service
   <img src="docs/assets/img/wrapper.png"/>
 </p>
 
+## Why you need lakeFS and what it can do
+
+lakeFS provides a Git-like branching and committing model that scales to Petabytes of data by utilizing S3 or GCS for storage.
+
+This branching model makes your data lake ACID compliant by allowing changes to happen in isolated branches that can be created, merged and rolled back atomically and instantly.
```

Comment:
Hi!

Thanks for your contribution.  We prefer to avoid the term "ACID".  It is heavily overloaded in the storage space and does not add clarity.  Could you either:
1. Define the term with references, and show how each of A-C-I-D meet the model, or:
2. Rephrase to avoid this term?

Thanks!